ProcessList.c

```diff
@@ -8,22 +8,28 @@ in the source distribution for its full text.
 #include "ProcessList.h"
 
 #include <assert.h>
+#include <stdlib.h>
 #include <string.h>
 #include <time.h>
 
 #include "CRT.h"
+#include "Hashtable.h"
+#include "Vector.h"
 #include "XUtils.h"
 
 
 ProcessList* ProcessList_init(ProcessList* this, const ObjectClass* klass, UsersTable* usersTable, Hashtable* pidMatchList, uid_t userId) {
    this->processes = Vector_new(klass, true, DEFAULT_SIZE);
```

Comment:
Should we also start with a Vector size of 200 (instead of DEFAULT_SIZE=10)?

Reply:
It's rare we have soo few processes on a system … Maybe increase `DEFAULT_SIZE`? But that's probably something for a follow-up patchset.